### ISSUE TITLE
Run in background

### ISSUE THREAD

**Issue by stevefan1999-personal** [nightmare-space/vscode_for_android#68]
Can we run this in the background so I can continue using it on my browser?

**Comment by mengyanshou** [comment]
> Can we run this in the background so I can continue using it on my browser?

sure, but u need set this app power mode(like MIUI)

**Comment by stevefan1999-personal** [comment]
> > Can we run this in the background so I can continue using it on my browser?
> 
> sure, but u need set this app power mode(like MIUI)

Yes I set it to unrestricted in Android but it still sleeps when I load another app. Wakelock is not working properly
